README.md

```diff
@@ -73,6 +73,28 @@ export LC_ALL=en_US.UTF-8
 export LANG=en_US.UTF-8
 ```
 
+### Docker
+
+Si tienes Docker instalado (https://docs.docker.com/install/) puedes hacer uso del Dockerfile en el directorio principal para crear una imagen y posteriormente ejectar un contenedor con todas las dependencias necesarias:
+
+```bash
+# En un sistema *nix
```

Comment:
En general estas instrucciones aplican para *nix y windows usando bash for windows

Reply:
Cierto, necesario que se añada?